### DISCUSSION TITLE
Export connection

### DISCUSSION THREAD

**Discussion by digiserg** [axonops/axonops-workbench#626] [Ideas]
On occasions I'm asked to share the connection details to one of the clusters I manage. I wonder if it would be a good idea to do something like this:

- Select the connection to export
- Optionally ask whether I want to include SSL certs
- Optionally also ask if I should include the credentials
- Finally, generate a secure file to share it. A `zip` with a password may not be enough, perhaps `gpg` encrypted with a passphrase.

The encrypted archive can then be shared with a colleague and he/she would be able to import it.

**Comment by millerjp** [comment]
@digiserg thats kind of what the workspace config is meant for. We are going to add a feature to import directly from (see #316 ) also #413 . Would that not be ok? Or is this just intended for adhoc sharing a specific connection and sharing a workspace might not be wanted?

**Comment by digiserg** [comment]
You would never store credentials in Git, whilst if we had a secure way of exporting the config into an encrypted bundle you could consider sharing the credentials and SSL certs.
